extensions/discourse_tagging.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module ::DiscourseTagging
+  class << self
+    alias_method :discourse_core_filter_allowed_tags, :filter_allowed_tags
```

Comment:
@fzngagan Any reason we're not using a module prepension and ``super`` here?

Reply:
I thought it wasn't possible but did some reading and experiments and yes, there's a way to do that. Will update the code accordingly.

Reply:
Fixed